File: src/main/java/hu/finominfo/sheduler/common/Globals.java
package hu.finominfo.sheduler.common;

/**
 * Created by kalman.kovacs@globessey.local on 2017.12.18.
 */
public class Globals {
    private static Globals ourInstance = new Globals();

    public static Globals getInstance() {
        return ourInstance;
    }

    private Globals() {
    }

    private final String configFile = "config.csv";
    private final String resultFile = "result.csv";

    public String getConfigFile() {
        return configFile;
    }

    public String getResultFile() {
        return resultFile;
    }
}

File: scheduler_python/common/globals.py
# scheduler_python/common/globals.py

class Globals:
    """
    A direct translation of the Java Globals class.
    In Java, it uses a singleton pattern. Here we can just have a simple class or constants.
    """
    def __init__(self):
        # Mirroring the Java file names:
        self.config_file = "config.csv"
        self.result_file = "result.csv"

    def get_config_file(self):
        return self.config_file

    def get_result_file(self):
        return self.result_file
